.buildkite/scripts/common.sh

```diff
@@ -12,6 +12,8 @@ ONLY_DOCS=${ONLY_DOCS:-"true"}
 OSS_MODULE_PATTERN="^[a-z0-9]+beat\\/module\\/([^\\/]+)\\/.*"
 XPACK_MODULE_PATTERN="^x-pack\\/[a-z0-9]+beat\\/module\\/([^\\/]+)\\/.*"
 # define if needed run the whole pipeline for the particular beat
+[ -z "${run_auditbeat+x}" ] && run_auditbeat="$(buildkite-agent meta-data get run_auditbeat --default "false")"
+[ -z "${run_filebeat+x}" ] && run_auditbeat="$(buildkite-agent meta-data get run_filebeat --default "false")"
```

Comment:
variable name is not matching the name of the beat

```suggestion
[ -z "${run_filebeat+x}" ] && run_filebeat="$(buildkite-agent meta-data get run_filebeat --default "false")"
```

Reply:
@alexsapran What step definitions?

Reply:
There is one here https://github.com/elastic/beats/pull/38605/files#diff-18f9ca90b8c92c28fd6c97266f3b8214cbf81cdb37ed36784930965cdc8bf13eR22 and one https://github.com/elastic/beats/pull/38605/files#diff-18f9ca90b8c92c28fd6c97266f3b8214cbf81cdb37ed36784930965cdc8bf13eR25

using the same `key: run_filebeat`

Reply:
If you mean inputs, not steps then it's fine

Reply:
Yes